packages/paste-website/src/component-examples/ModalExamples.ts

```diff
@@ -238,3 +238,48 @@ render(
   <ModalTrigger />
 )
 `.trim();
+
+export const badgeModalExample = `
+const ModalTrigger = () => {
+  // Modal properties
+  const [isOpen, setIsOpen] = React.useState(false);
+  const handleOpen = () => setIsOpen(true);
+  const handleClose = () => setIsOpen(false);
+  const modalHeadingID = useUID();
+
+  return (
+    <div>
+      <Badge as="button" variant="new" onClick={handleOpen}>
+        <NewIcon decorative/>
+        beta
+      </Badge>
+      <Modal ariaLabelledby={modalHeadingID} isOpen={isOpen} onDismiss={handleClose} size="default">
+        <ModalHeader>
+          <ModalHeading as="h3" id={modalHeadingID}>
+            Beta mode
+          </ModalHeading>
+        </ModalHeader>
+        <ModalBody>
+
+          <Paragraph>
+            This product is in beta, which means we're still working out all the kinks. Let us know if you spot any bugs.
+          </Paragraph>
+
+        </ModalBody>
+        <ModalFooter>
+          <ModalFooterActions>
+            <Button variant="secondary" onClick={handleClose}>
+              Cancel
+            </Button>
+            <Button variant="primary">Done</Button>
+          </ModalFooterActions>
```

Comment:
V nit picky, but can we just have a single "Got it" button in the footer, instead of "Cancel" and "Done"?

Reply:
No problem!

Reply:
u da best